src/main/java/com/simibubi/create/content/kinetics/chainConveyor/ChainConveyorRenderer.java

```diff
@@ -82,6 +100,12 @@ private void renderBox(ChainConveyorBlockEntity be, PoseStack ms, MultiBufferSou
 
 		Vec3 position = physicsData.prevPos.lerp(physicsData.pos, partialTicks);
 		Vec3 targetPosition = physicsData.prevTargetPos.lerp(physicsData.targetPos, partialTicks);
+		if (frustum != null && !frustum.testSphere(
```

Comment:
I strongly disagree with the per-box frustum tests when you still have not provided any guarantee that the unit bounding sphere will always be a conservative approximation for 3rd party package models